docker/theia-app-theia-ext/README.md

```diff
@@ -0,0 +1,48 @@
+# Example docker image for theia-trace-extension
+
+This folder contains an example showing how to create a docker image, that bundles at its core a Theia-based application that uses the _Theia Trace Viewer_ front-end.
+
+Notes:
+
+- the image will contain exclusively the trace viewer front-end.
+  If you want to run a complete _trave viewer_ application, you will need a service running the trace-server (not included here);
+
+- the image will use a pre-built, specific version of the _theia-traceviewer_ extension [npm package]
+
+## How to build and run
+
+Build the image and name it _tate_ (short for "theia application - theia extension"). You may use a different name if you prefer.
+
+Try adding `--network host` in case of build failures related to debian packages retrieval:
+
+```bash
+docker build -t tate .
+```
+
+Once the image has been built, start a container named _tate-1_ from
+the _tate_ image:
+
+```bash
+docker run --name tate-1 tate
```

Comment:
After building the images from the parent directory, I went to this directory and and tried this command. I get the following error message:
```
Unable to find image 'tate:latest' locally
docker: Error response from daemon: pull access denied for tate, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
```

Reply:
The faulty instructions have been migrated to the parent folder README and adjusted. Can you please retry?